post_installation_configuration/coreos-layering.adoc

```diff
@@ -84,6 +84,13 @@ RUN rpm-ostree install libreswan && \
 Use the same base {op-system} image installed on the rest of your cluster. Use the `oc adm release info --image-for rhel-coreos-8` command to obtain the base image used in your cluster.
 ====
 
+[NOTE]
+====
+Some packages require additional configuration options, one common example is the kernel. When replacing the kernel you will need to enable rpm-ostree's cliwrap by running: 
+`rpm-ostree cliwrap install-to-root /`
+on your Containerfile. More examples can be seen on link:https://github.com/coreos/layering-examples/blob/main/replace-kernel/Containerfile[CoreOS layering examples repository.]
```

Comment:
I think we were narrowing in on dropping this requirement, right?  But yes, let's link to there for now.

Reply:
True, but it's in the current 4.12 image. Someone hit this today.

Reply:
Yeah.  Hmm.  I think probably what we should do is copy at least that kernel override Dockerfile into this document.  That way it will naturally be versioned with the OCP release

Reply:
I will add the kernel example here.